generators/server/templates/_gradle.properties

```diff
@@ -80,8 +80,8 @@ spring_security_oauth2_version=2.0.12.RELEASE
 <%_ } _%>
 springfox_version=2.7.0
 spring_boot_version=1.5.7.RELEASE
-<%_ if (serviceDiscoveryType || applicationType === 'gateway' || applicationType === 'microservice' || applicationType === 'uaa') { _%>
-spring_cloud_version=Dalston.SR2
+<%_ if (serviceDiscoveryType ||\u00c2\u00a0applicationType === 'gateway' || applicationType === 'microservice' || applicationType === 'uaa') { _%>
```

Comment:
Something happened to your spaces here before `applicationType`

Reply:
Did not see the wrong encoded character.
I fixed it!
Thanks